zebrad/tests/acceptance.rs

```diff
@@ -1835,7 +1835,7 @@ fn lightwalletd_integration_test(test_type: TestType) -> Result<()> {
 
         if test_type.needs_lightwalletd_cached_state() {
             lightwalletd
-                .expect_stdout_line_matches("Done reading [0-9]{1,7} blocks from disk cache")?;
+                .expect_stdout_line_matches("Done reading [0-9]{7} blocks from disk cache")?;
```

Comment:
Some tests in this PR will fail until we run the fixes below in CI. But that's good, because when those tests pass, that tells us we've fixed the cached state and can merge this PR.

Reply:
Before we do the new step 2, we'll need to revert commit [`0fb1093` (#7507)](https://github.com/ZcashFoundation/zebra/pull/7507/commits/0fb10938d9903d59c338b93a6fad4911aa2d969e).

Sorry about the extra work, sometimes it takes a while to think of shorter alternatives.

Reply:
I just pushed a commit that reverts this temporary change.